products/log4j.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Log4j
+permalink: /log4j
+layout: post
+category: server-app
+link: https://logging.apache.org/log4j/2.x/security.html
+changelogTemplate: https://logging.apache.org/log4j/2.x/changes-report.html#a{{"__LATEST__"}}
+activeSupportColumn: false
+releaseDateColumn: true
+iconSlug: apache
+sortReleasesBy: 'releaseCycle'
+releases:
+  - releaseCycle: "2"
```

Comment:
It is important to show those version numbers so i suggest we divide those ones in new releaseCycles like 2.X , 2.12.X  and 2.3.X . I know that 2.X should cover all below but it will let us show new comers that it is already reached EOL